app/models/parameter.rb

```diff
@@ -39,6 +39,7 @@ class Parameter < ApplicationRecord
   default_scope -> { order("parameters.name") }
 
   before_create :set_priority
+  before_save :set_cloned_value
```

Comment:
I think we should add this column the ones that aren't audited, since it's an internal attribute only used for search